graders/c/Dockerfile

```
@@ -19,8 +19,8 @@ ENV LC_LANG=en_US.UTF-8
 ENV PYTHONIOENCODING=UTF-8
 ENV PYTHONPATH=/cgrader/:/grade/serverFilesCourse
 
-RUN groupadd sbuser
-RUN useradd -g sbuser sbuser
+RUN groupadd sbuser && \
+    useradd -g sbuser sbuser
```

Nit, may as well collapse this to one line:

```suggestion
RUN groupadd sbuser && useradd -g sbuser sbuser
```

---

Fixed in 5ae834225dd4c7b110bee80a810331ddd36ee375.